pkg/api/entries.go

```diff
@@ -301,9 +305,64 @@ func createLogEntry(params entries.CreateLogEntryParams) (models.LogEntry, middl
 	logEntry := models.LogEntry{
 		entryID: logEntryAnon,
 	}
+
+	if api.newEntryPublisher != nil {
+		// Publishing notifications should not block the API response.
+		go func() {
+			var subjects []string
+			verifiers, err := entry.Verifiers()
+			if err != nil {
+				log.ContextLogger(ctx).Warnf("Could not get verifier for log entry %s: %w", entryID, err)
```

Comment:
If you can't get the Verifiers for an entry, that feels like more than just a warning to me. Curious to get @haydentherapper 's opinion.

Reply:
Agreed - It would mean that the entry or verifier is malformed, since it should always contain a valid verifier. We should log an error/metric and return.